web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -667,6 +682,8 @@
                 .toLowerCase()
                 .endsWith('.insp'))}
             <PanoramaViewer {asset} />
+          {:else if $isShowEditor && selectedEditType == 'crop'}
```

Comment:
Please use strict equality by default

```suggestion
          {:else if $isShowEditor && selectedEditType === 'crop'}
```